upup/pkg/fi/fitasks/managedfile.go

```diff
@@ -43,8 +43,9 @@ type ManagedFile struct {
 
 	Contents fi.Resource
 
-	// Public controls whether the object is world-readable
-	Public *bool
+	// PublicACL controls whether the _object_ has an ACL which grants world-readable status.
+	// Note that the _bucket_ may itself have a grant for world-readable, that is separate.
```

Comment:
```suggestion
	// Note that the _bucket_ may itself have a grant for world-readable; that is separate.
```

Reply:
Moved this to https://github.com/kubernetes/kops/pull/14775 to reduce the number of touched by this PR.